packages/cloudflare/src/cli/templates/init.ts

```diff
@@ -61,6 +61,9 @@ function initRuntime() {
 
   globalThis.__dirname ??= "";
   globalThis.__filename ??= "";
+  // Some packages rely on `import.meta.url` but it is undefined in workerd
+  // For example it causes a bunch of issues, and will make even import crash with payload
+  import.meta.url ??= "file:///worker.js";
```

Comment:
```suggestion
  import.meta.url ??= "file://worker.js";
```

is this intentional?

Reply:
Yeah that's the default file uri scheme